README.md

```diff
@@ -27,12 +27,35 @@ list(APPEND CMAKE_PREFIX_PATH ${CMAKE_BINARY_DIR})
 
 add_definitions("-std=c++11")
 
+set (CONAN_CMAKE_VERSION "0.17.0")
```

Comment:
Note: This & the logic below assume that the format of the file version will be the same going forward within the file _and_ in the file download URL.  This historically has not always been the case.  Version 0.16 appears as "0.16.0" in the file, but "0.16" in the URL.